### ISSUE TITLE
Fix for IDISP006 implements IDisposable but doesn't dispose any fields

### ISSUE THREAD

**Issue by jnm2** [DotNetAnalyzers/IDisposableAnalyzers#101]
When invoking the 'Implement IDisposable' fix on either field, the fix could go ahead and dispose all the fields that currently end up with IDISP002 'Dispose member.'

```cs
class Repro
{
    private Disposable disposable1 = new Disposable();
    private Disposable disposable2 = new Disposable();
}
```

**Comment by JohanLarsson** [comment]
Yes, this is very reasonable. Wrote it like this to see that the steps were correct in the beginning and never updated it.

**Comment by JohanLarsson** [comment]
I'm starting to think maybe we should wait with this. Reason is that if we dumb analysis in some other place things that should not be disposed may be disposed. By doing it field by field there is a better chance to see if the analyzer dumbs it. In the repro class it would be obvious but in a poorly formatted class with 1000 lines it will not be obvious. The issue is good so we leave it open of course!
